php.ini-development

```diff
@@ -1117,71 +1123,70 @@ odbc.allow_persistent = On
 ; https://php.net/odbc.check-persistent
 odbc.check_persistent = On
 
-; Maximum number of persistent links.  -1 means no limit.
+; Maximum number of persistent links. -1 means no limit.
 ; https://php.net/odbc.max-persistent
 odbc.max_persistent = -1
 
-; Maximum number of links (persistent + non-persistent).  -1 means no limit.
+; Maximum number of links (persistent + non-persistent). -1 means no limit.
 ; https://php.net/odbc.max-links
 odbc.max_links = -1
 
-; Handling of LONG fields.  Returns number of bytes to variables.  0 means
+; Handling of LONG fields. Returns number of bytes to variables. 0 means
 ; passthru.
 ; https://php.net/odbc.defaultlrl
 odbc.defaultlrl = 4096
 
-; Handling of binary data.  0 means passthru, 1 return as is, 2 convert to char.
+; Handling of binary data. 0 means passthru, 1 return as is, 2 convert to char.
 ; See the documentation on odbc_binmode and odbc_longreadlen for an explanation
 ; of odbc.defaultlrl and odbc.defaultbinmode
 ; https://php.net/odbc.defaultbinmode
 odbc.defaultbinmode = 1
 
-[MySQLi]
-
-; Maximum number of persistent links.  -1 means no limit.
+[mysqli]
+; Maximum number of persistent links. -1 means no limit.
 ; https://php.net/mysqli.max-persistent
 mysqli.max_persistent = -1
 
 ; Allow accessing, from PHP's perspective, local files with LOAD DATA statements
 ; https://php.net/mysqli.allow_local_infile
-;mysqli.allow_local_infile = On
+;mysqli.allow_local_infile = 1
```

Comment:
Does this have more options than On/Off?

Reply:
According to the [documentation](https://www.php.net/manual/en/mysqli.configuration.php#ini.mysqli.allow-local-infile) this is int.

Reply:
Ditto, documentation is not the source of truth.

Reply:
https://github.com/php/php-src/blob/f90323c8d4456ff2b9a0f442f6a383d9b58185ae/ext/mysqli/mysqli.c#L443